.github/workflows/ci.yml

```diff
@@ -11,6 +11,10 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
+defaults:
+  run:
+    shell: bash
```

Comment:
Why do we need this

Reply:
For using bash in windows
https://github.com/Watfaq/clash-rs/pull/913/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fL255-L260